recipes/libgpuarray/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: libgpuarray
+  version: "0.6.0"
+
+source:
+  git_url: https://github.com/Theano/libgpuarray.git
+  git_tag: v0.6.0
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+    - vc14  # [win and py36]
+
+requirements:
+  build:
+    - git
+    - cmake
+    - python
+    - cython >=0.25
+    - numpy
+    - mako
+    - setuptools
```

Comment:
I think numpy is required for the headers since I build cython modules that use them.  The rest are all used during the build.

If we don't need numpy for cython modules, then I can remove this.